gdmix-workflow/README.md

```diff
@@ -48,7 +48,7 @@ We'll also need a GDMix config, a reference of training logistic regression mode
 Please see the section [Try out the movieLens example](../README.md#Try-out-the-movieLens-example) in the root [README.md](../README.md) for details of how to run the movieLens example on single node.
 
 ### Run on Kubernetes for distributed training
-To run on Kubernetes, as mentioned earlier, user will need to copy the processed movieLens data to the centralized storage, modify the input path fields such as `train_data_path`,  `validation_data_path`, `feature_file` and `metadata_file` of the GDMix config for distributed training [lr-distributed-movieLens.config](examples/movielens-100k/lr-distributed-movieLens.config).
+To run on Kubernetes, as mentioned earlier, user will need to copy the processed movieLens data to the centralized storage, modify the input path fields such as `training_data_dir`,  `validation_data_dir`, `feature_file` and `metadata_file` of the GDMix config for distributed training [lr-distributed-movieLens.config](examples/movielens-100k/lr-distributed-movieLens.config).
```

Comment:
do you want to bump major version if it's not 0 or minor version for backward-incompatible changes and document those?

Reply:
We haven't released 1.x yet, because we're in the process of migrating to TF2, we may bump version then? @junshi15 What do you think?

Reply:
let's release 1.0 when TF2 is ready. Currently we are still on 0.x

Reply:
Version has been bumped from 0.2.1 to 0.3 in this PR.